code/game/objects/items/tools/powertool.dm

```diff
@@ -0,0 +1,86 @@
+//Power Tools
+/obj/item/powertool
+	var/stateswap = 0 //what state is it in
+	var/icon_state_swap = 0 //which icon should it be
+	var/tool_behaviour_swap = 0 //what tool behaviour(s) should be here
+	var/swapsound = 0 //the sound that plays for swapping
+
+/obj/item/powertool/attack_self(mob/living/carbon/user) //activating the item causes it to swap icons and behaviours
+	stateswap = !stateswap
+	if(stateswap)
+		icon_state = icon_state_swap
+		tool_behaviour = tool_behaviour_swap
+	else
+		icon_state = initial(icon_state)
+		tool_behaviour = initial(tool_behaviour)
+	playsound(get_turf(user), swapsound, 50, 1)
+	to_chat(user, "<span class='notice'>You attach a different bit to [src].</span>")
+	return
+		
+/obj/item/powertool/drill
+	name = "Power Drill"
+	desc = "A simple hand drill with bolt and screw bits."
+	icon = 'icons/obj/tools.dmi'
+	icon_state = "drill_screw"
+	item_state = "drill"
+	icon_state_swap = "drill_bolt"
+	hitsound = 'sound/items/drill_hit.ogg'
+	usesound = 'sound/items/drill_use.ogg'
+	swapsound = 'sound/items/change_drill.ogg'
+	flags = CONDUCT
+	slot_flags = SLOT_BELT
+	force = 5
+	throwforce = 7
+	w_class = WEIGHT_CLASS_SMALL
+	materials = list(MAT_METAL=150,MAT_SILVER=50,MAT_TITANIUM=25)
```

Comment:
Nit: space after commas (same below).
I know you've simply moved this from another place, but this is inconsistent with the other lists in here if nothing else.
(feel free to ignore)